install_conda_deps.sh

```diff
@@ -4,4 +4,6 @@ conda install -n $YOURENVNAME requests=2.13.0
 conda install -n $YOURENVNAME -c menpo opencv=2.4.11
 conda install -n $YOURENVNAME pyqt=5.6.0
 conda install -n $YOURENVNAME qtawesome=0.4.4
+conda install -n $YOURENVNAME nomkl numpy scipy scikit-learn numexpr
+conda remove -n $YOURENVNAME mkl mkl-service
```

Comment:
Sounds good, what about numpy, scipy and scikit-learn having their own version numbers? If nothing else we should at least specify a major version number so we don't have unexpected functionality changes in the futur.